New Agent troubleshooting section (#4606)
DataDog/documentation: content/en/agent/troubleshooting/agent_check_status.md

---
title: Agent Check Status
kind: documentation
further_reading:
- link: "/agent/troubleshooting/debug_mode"
  tag: "Agent Troubleshooting"
  text: "Agent Debug Mode"
- link: "/agent/troubleshooting/send_a_flare"
  tag: "Agent Troubleshooting"
  text: "Send an Agent Flare"
---

> If you are experiencing issues with a given Agent Check, find below the list of commands for your OS in order to get more troubleshooting information:

Reviewer: ```suggestion
If you are experiencing issues with a given Agent Check, use these commands for your OS to get more troubleshooting information:
```

- [Linux](#linux)
- [Windows](#windows)
- [Systemd](#systemd)
- [Further Reading](#further-reading)

## Linux

> Agent checks must be called by the Agent. To test an Agent check, run:

Reviewer: ```suggestion
To test an Agent check, run:
```
